can/interfaces/pcan/pcan.py

```diff
@@ -27,8 +32,8 @@
     timeout_clock = time.clock
 
 # Set up logging
-logging.basicConfig(level=logging.WARNING)
 log = logging.getLogger('can.pcan')
+log.setLevel(logging.WARNING)
```

Comment:
IMO we shouldn't be setting the logging level at all outside of our scripts. So :+1: for removing the `basicConfig` but I'd say also remove the `setLevel` call.